packages/runner-ct/src/specs/specs-list.tsx

```diff
@@ -1,43 +1,49 @@
+import * as React from 'react'
 import { observer } from 'mobx-react'
-import React, { Component } from 'react'
 import State from '../lib/state'
 import { makeSpecHierarchy } from './make-spec-hierarchy'
 import { SpecItem } from './spec-item'
 
 interface SpecsListProps {
   state: State
+  config: Cypress.ConfigOptions
 }
 
-@observer
-class SpecsList extends Component<SpecsListProps> {
-  render () {
-    const { state } = this.props
-
+export const SpecsList: React.FC<SpecsListProps> = observer(
+  function SpecsList ({ state, config }) {
     const specGroups = state.filteredSpecs.length ? makeSpecHierarchy(state.filteredSpecs) : []
 
     return (
       <div className="specs-list">
         <header>
+          <h1>
+            { // @ts-expect-error config.project name is not typed
+              config.projectName ?? document.title ?? 'Cypress'
```

Comment:
I have noticed a *lot* of things are not typed on Cypress.ConfigOptions. I think ConfigOptions is only for *private* APIs? :thinking:

Reply:
It is the private API